web/src/components/vis/AnovaVolcanoPlotTile.vue

```diff
@@ -136,11 +136,16 @@ vis-tile-large(v-if="dataset", title="Metabolite Anova Volanco Plot", :loading="
     metabolite-colorer(:dataset="dataset", v-model="metaboliteColor",
         empty-option="No Color")
 
+  // can refactor v-if to use optional chaining when viime is updated to Vue 3
```

Comment:
Can you file an issue to describe this requested change, and maybe recast this comment as a TODO?

Reply:
https://github.com/girder/viime/issues/633